docker-compose.yml

```diff
@@ -2,16 +2,45 @@ version: '3.8'
 services:
   mysql:
     image: mysql:8.0
-    command: --default-authentication-plugin=mysql_native_password
+    restart: always
+    command:
+      --default-authentication-plugin=mysql_native_password
+      --bind-address=0.0.0.0
```

Comment:
it can be removed since Docker already isolates and manages network access to your containers. When another container wants to connect to your MySQL container, it will do so through the Docker network, not directly through an IP address.